### DISCUSSION TITLE
Sync collections between platforms

### DISCUSSION THREAD

**Discussion by valentino-90** [thunderclient/thunder-client-support#994] [Ideas]
Hi there!
Amazing extension. Been using for a while, thanks so much!

I have a question, is there a possibility that in the future to sync collections between vscode under different machines (same github account) ?

Thanks a lot again.

**Comment by rangav** [comment]
@valentino-90 Glad you like the extension.

You can save request data to git workspace.

see docs here
https://github.com/rangav/thunder-client-support#git-sync
